### PR TITLE
feat: daily.dev plus MVP

### PR DESCRIPTION
**Pull request by rebelchris**
## Changes

First bit of the plus page

Note:
This is only the first introduction, added separate tasks for tracking events etc.
@omBratteng Specially for you good to review/test locally for the rest of development

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://as-697-plus-mvp.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/plus/PlusList.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../typography/Typography';
+import { IconSize } from '../Icon';
+
+const PlusItem = ({ title }: { title: string }) => {
+  return (
+    <li className="flex gap-2">
+      <ChecklistBIcon className="text-text-tertiary" size={IconSize.Small} />{' '}
```
Comment (omBratteng) [inline packages/shared/src/components/plus/PlusList.tsx:14]:
```suggestion
      <ChecklistBIcon className="text-text-tertiary" size={IconSize.Small} />
```
This empty space needed?

File: packages/shared/src/contexts/PaymentContext.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  const getPrices = useCallback(() => {
+    return paddle
+      ?.PricePreview({
+        items: Object.keys(planTypes).map((priceId) => ({
+          priceId,
+          quantity: 1,
+        })),
+      })
+      .then((response) => response);
+  }, [paddle, planTypes]);
```
Comment (omBratteng) [inline packages/shared/src/contexts/PaymentContext.tsx:89]:
```suggestion
  const getPrices = useCallback(async () => {
    return paddle?.PricePreview({
      items: Object.keys(planTypes).map((priceId) => ({
        priceId,
        quantity: 1,
      })),
    });
  }, [paddle, planTypes]);
```

File: packages/shared/src/contexts/PaymentContext.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+  const { data: productPrices } = useQuery({
+    queryKey: ['productPrices'],
+    queryFn: async () => {
+      return getPrices();
+    },
```
Comment (omBratteng) [inline packages/shared/src/contexts/PaymentContext.tsx:95]:
```suggestion
    queryFn: async () => getPrices(),
```

Could do this too, though not blocker

File: packages/shared/src/graphql/paddle.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+export const PADDLE_PRICING_IDS_QUERY = gql`
+  query PricingIds {
+    pricingIds {
+      value
+    }
+  }
+`;
+
+export const getPricingIds = async (): Promise<EmptyResponse> => {
```
Comment (omBratteng) [inline packages/shared/src/graphql/paddle.ts:13]: Return type correct?

File: packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React, { ReactElement, ReactNode } from 'react';
+import { MainFeedPageProps } from '../MainFeedPage';
+import { PlusHeader } from './PlusHeader';
+
+export default function PlusLayout({
+  children,
+}: MainFeedPageProps): ReactElement {
+  return (
+    <main className="flex h-screen flex-col bg-blur-glass before:absolute before:inset-0 before:-z-1 before:bg-[url('https://daily-now-res.cloudinary.com/image/upload/s--q2q8AL3Q--/f_auto/v1731397654/Upgrade_to_plus_i8phhr')] before:content-['']">
```
Comment (omBratteng) [inline packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx:9]: Use absolute img? And then move image to cloudinary constants?

Comment (rebelchris) [reply packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx:9]: I was too lazy to move the flex order around 😂
We can do it yeah, probably should.

Also the image is not correct now I saw.

Comment (omBratteng) [reply packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx:9]: We could always add the background images to tailwind 😬

File: packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  children,
+}: MainFeedPageProps): ReactElement {
+  return (
+    <main className="relative flex h-screen flex-col bg-blur-glass">
+      <img
+        src={cloudinaryPlusBackground}
+        alt="Plus background"
+        className="absolute inset-0 -z-1 h-full w-full"
+        aria-hidden
+      />
```
Comment (omBratteng) [inline packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx:16]: Should we create a global component for these sort of images?
Non-blocking

File: packages/shared/src/lib/image.ts [hunk 7]
```diff
@@ -258,6 +258,8 @@ export const cloudinaryIntegrationsSlackHeaderDark =
 
 export const cloudinaryTopReaderBadgeBackground =
   'https://media.daily.dev/image/upload/v1730888952/webapp/topReaderBadgeBackground.svg';
+export const cloudinaryPlusBackground =
+  'https://daily-now-res.cloudinary.com/image/upload/s--q2q8AL3Q--/f_auto/v1731397654/Upgrade_to_plus_i8phhr';
```
Comment (omBratteng) [inline packages/shared/src/lib/image.ts:262]: Was this the correct image?

File: packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx (hunk 6 above)
Comment (rebelchris) [reply packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx:16]: Let's add it to backlog :)

File: packages/shared/src/lib/image.ts (hunk 7 above)
Comment (rebelchris) [reply packages/shared/src/lib/image.ts:262]: No but don't want to block merge to intermediate for a image.

Comment (omBratteng) [reply packages/shared/src/lib/image.ts:262]: 👍

File: packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx (hunk 6 above)
Comment (omBratteng) [reply packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx:16]: 👍

**Review by omBratteng — APPROVED** [review]
Code looks fine and in preview

**Pull request merged**